.travis.yml

```diff
@@ -7,7 +7,27 @@ services:
 before_script:
   - docker --version
 script:
+  - sudo mkdir /compiled # directory to store compiled library
   - cd libamtrackForJs
-  - docker-compose build
-# - if build form other branch or from fork changes params below and use it or in docker-compose.yml
-# - docker build --build-arg repoUrl=https://github.com/libamtrack/library.git --build-arg branch=forjs .
\ No newline at end of file
+
+  # compile with WASM support
+  - docker build --build-arg repoUrl=https://github.com/libamtrack/library.git --build-arg branch=forjs --build-arg WASM=1 -t libamtrack/libamtrackforjs:latest .
+  - docker run -itd --name libamtrack libamtrack/libamtrackforjs
+  - sudo docker cp libamtrack:/libat.js /compiled
```

Comment:
Does docker needs sudo ?
https://docs.travis-ci.com/user/docker/

Reply:
No it dont, but copy to /compiled directory requieres it.

Reply:
We can use any directory is is just to store it until we colne web repo

Reply:
Try using some other directory, relative to `pwd` - then sudo may not be needed: copying 
 to `/compiled` looks like copying to some system top level directory (/etc/, dev/)